src/util/FFstrbuf.c

```diff
@@ -29,6 +29,14 @@ void ffStrbufInitCopy(FFstrbuf* strbuf, const FFstrbuf* src)
     ffStrbufAppend(strbuf, src);
 }
 
+void ffStrbufInitS(FFstrbuf* strbuf, const char* str)
+{
+    uint32_t bufSize = (uint32_t)strlen(str) + 1;
+    ffStrbufInitA(strbuf, FASTFETCH_STRBUF_DEFAULT_ALLOC > bufSize ? FASTFETCH_STRBUF_DEFAULT_ALLOC : bufSize);
+    memcpy(strbuf->chars, str, bufSize);
+    strbuf->length = bufSize - 1;
```

Comment:
Is this really faster than an init and an append?

Reply:
Ok, but isn't this a bug?

https://github.com/LinusDierheimer/fastfetch/blob/master/src/util/FFstrbuf.c#L100

Reply:
https://github.com/LinusDierheimer/fastfetch/blob/master/src/util/FFstrbuf.c#L112

Shouldn't be `strbuf.length + length + 1`?

Reply:
No, `ffStrbufEnsureFrees` ensures that there is enough space to append the given length, including the null byte.